content/blog/2022/100-percent-test-coverage.md

```diff
@@ -0,0 +1,157 @@
+---
+title: What's the impact of aiming for 100% test coverage?
+author: olu
+date: 2022-05-24
+tags:
+  - Article
+  - Testing
+  - Test Coverage
+image:
+  src: blog/2022/hundred-g8323bcdec_1920.jpg
+  alt: Red "one hundred percent" repeated on a white background.
+summary: |
+  100% test coverage is a contentious metric! In this piece, Olu explores the
+  impact of pursuing it.
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+Test coverage is a way of measuring how much of the codebase is covered by
+tests. It usually refers to unit tests or integration tests. There are automated
+ways of measuring it, and these usually present your coverage as a percentage.
+100% test coverage is a contentious issue. At OddBird we often aim for this
+level of test coverage, and after working on a few different projects I thought
+it would be good to assess how I find it and the pros and cons of testing in
+this way.
+
+{{ embed.img(
+  src='blog/2022/metecho-coverage.png',
+  alt='One hundred percent test coverage on the Metecho project'
```

Comment:
Maybe we could add some additional description to the alt text of what the image shows? Something like: `Image of the Metecho project's test coverage summary which shows 100% test coverage for statements, branches, functions and lines. Also displayed is the number of test suites, tests, snapshots and the time it took to run the tests`